tests/PyPI-pytest/run_all.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Python test originally created or extracted from other peoples work. The
+#     parts from me are licensed as below. It is at least Free Software where
+#     it's copied from other people. In these cases, that will normally be
+#     indicated.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+
+import os
+import sys
+import subprocess
+
+from nuitka.tools.testing.OutputComparison import compareOutput
+from nuitka.tools.testing.Virtualenv import withVirtualenv
+from nuitka.utils.FileOperations import removeDirectory
+
+packages = (
+    "urllib3",
+)
+
+base_dir = os.getcwd()
+
+
+
+
+for package_name in packages:
+    try:
+        # get normal pytest results
+        with withVirtualenv("venv_%s" % package_name) as venv:
+            venv.runCommand(
+                commands=[
+                "call Scripts/activate",
+                "git clone https://github.com/%s/%s.git" % (package_name,package_name),
+                "python -m pip install pytest",
+                "cd %s" % package_name,
+                "python -m pip install -r dev-requirements.txt",
+                # "python -m pytest --disable-warnings",
```

Comment:
`python -m pytest --disable-warnings` (currently commented out) runs the normal pytest correctly, but the output is directed to the shell and I don't know of a way to redirect it

Reply:
and I need to save it as a string in some way to be able to compare it to the output of the nuitka wheel pytest